src/pages/docs.tsx

```diff
@@ -165,9 +171,395 @@ function renderArticleSections(
   return sections;
 }
 
-function renderArticle(page: ApiDocsObj | null): JSX.Element {
+function renderArticle(
+  page: ApiDocsObj | null,
+  userOS: UserOS,
+  version: string
+): JSX.Element {
+  const sourceCodeLink = downloadUrlByOs(UserOS.UNIX, version);
+  const prebuiltInstallerLink = downloadUrlByOs(userOS, version);
+
   if (!page) {
-    return <article>No Page Found</article>;
+    return (
+      <article className="article-reader">
+        <p className="t-overline">
+          <span>home</span>
+          &nbsp;&#47;&nbsp;
```

Comment:
use scss instead of nonbreaking spaces here

Reply:
fixed in f257538